docs/evaluate/temporal-cloud/gcp-private-service-connect.mdx

```diff
@@ -17,4 +17,80 @@ tags:
+[GCP Private Service Connect](https://cloud.google.com/vpc/docs/private-service-connect) allows you to open a path to Temporal without opening a public egress.
+It establishes a private connection between your Google Virtual Private Cloud (VPC) and Temporal Cloud.
+This one-way connection means Temporal cannot establish a connection back to your service.
+This is useful if normally you block traffic egress as part of your security protocols.
+If you use a private environment that does not allow external connectivity, you will remain isolated.
+
+Set up Private Service Connect with Temporal Cloud with these steps:
+
+1. Open the Google Cloud console
+2. Navigate to _Network Services_, then _Private Service Connect_. If you haven't used _Network Services_ recently, you might have to find it by clicking on _View All Products_ at the bottom of the left sidebar.
+
+   ![GCP console showing Network Services, and the View All Products button](/img/gcp-private-service-connect/gcp-console.png)
+
+3. Go to the _Endpoints_ section. Click on _Connect endpoint_.
+
+   ![GCP console showing the endpoints, and the Connect endpoint button](/img/gcp-private-service-connect/connect-endpoint-button.png)
+
+4. Under _Target_, select _Published service_, this will change the contents of the form to allow you to fill the rest as described below
+
+   ![GCP console showing the endpoints, and the Connect endpoint button](/img/gcp-private-service-connect/connect-endpoint.png)
+
+  - For _Target service_, fill in the _Service name_ with the Private Service Connect Service Name for the region you’re trying to connect to:
+
+      | Region                 | Private Service Connect Service Name                           |
+      | ---------------------- | -------------------------------------------------------------- |
+      | `us-central1`          | `projects/TBD`                                                 |
+      | `australia-southeast1` | `projects/TBD`                                                 |
+
+  - For _Endpoint name_, enter a unique identifier to use for this endpoint. It could be for instance `temporal-api` or `temporal-api-<namespace>` if you want a different endpoint per namespace.
+  - For _Network_ and _Subnetwork_, choose the network and subnetwork where you want to publish your endpoint.
+  - For _IP address_, click the dropdown and select _Create IP address_ to create an internal IP from your subnet dedicated to the endpoint. Select this IP.
+  - Check _Enable global access_ if you intend to connect the endpoint to virtual machines outside of the selected region. We recommend regional connectivity instead of global access, as it can be better in terms of latency for your workers.
+
+5. Click the _Add endpoint_ button at the bottom of the screen.
+   If successful, the status of your new endpoint will appear as _Accepted_.
+   Take note of the _IP address_ that has been assigned to your endpoint, as it will be used to connect to Temporal Cloud.
```

Comment:
Ultimately, we'll probably want to suggest people to set-up DNS for this, knowing that Google's product is botched for the DNS handling

Reply:
How in-depth would you want this? Should we just say something like `We recommend configuring DNS for this` or do you want us to link to some GCP specific docs?

Reply:
I think we're fine keeping this like this for now, since people can use the IP address directly.
In the future, we might want the customers to override `api.temporal.io` using a private DNS zone, but we're not there yet, so I think we can just keep it as it is phrased here for now